src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const basicPrice = 40;
+  const THREE_DAYS_DISC = 20;
+  const THREE_DAYS = 3;
```

Comment:
```suggestion
  const SHORT_TERM = 3;
```
Don't hardcode the numbers in variables' names. What if we need to change the term for small discount?
Fix all cases